src/app/shared/issue-tables/dropdownfilter.ts

```diff
@@ -45,5 +47,6 @@ export function applyDropdownFilter(dropdownFilter: DropdownFilter): (a: Issue)
     ret = ret && dropdownFilter.milestones.some((milestone) => issue.milestone.number === milestone);
 
     return ret && dropdownFilter.labels.every((label) => issue.labels.includes(label));
-  };
+  });
+  return getSortedData(dropdownFilter.sort, filteredData);
```

Comment:
Makes sense to apply filter then sort the data, but `applyDropdownFilter` only has one consumer in `IssuesDataTable::loadIssues` and `getSortedData` only has one consumer in `applyDropdownFilter`. Can we ensure single responsibility by just having `applyDropdownFilter` apply the filters, and after the filters have been applied, apply the sort directly in `IssuesDataTable::loadIssues`?

Reply:
I see what you mean. I have applied the recommended changes to perform sorting in ``IssuesDataTable::loadIssues``.